docs/feature_space_cadet_shift.md

```diff
@@ -24,14 +37,22 @@ COMMAND_ENABLE = no
 ## Configuration
 
 By default Space Cadet assumes a US ANSI layout, but if your layout uses different keys for parentheses, you can redefine them in your `config.h`.
-You can also disable the rollover, allowing you to use the opposite Shift key to cancel the Space Cadet state in the event of an erroneous press, instead of emitting a pair of parentheses when the keys are released.
+You can also disable the rollover, allowing you to use the opposite modifer key to cancel the Space Cadet state in the event of an erroneous press, instead of emitting a pair of parentheses when the keys are released.
```

Comment:
```suggestion
You can also disable the rollover, allowing you to use the opposite modifier key to cancel the Space Cadet state in the event of an erroneous press, instead of emitting a pair of parentheses when the keys are released.
```